docs/VisualizingECL/VizMods/VizIntro.xml

```diff
@@ -0,0 +1,146 @@
+    platform to allow you to create visualizations from the results from
+    queries written in ECL.</para>
+
+    <para>To install, use the ecl command line interface.</para>
+
+    <para></para>
+
+    <para><programlisting>ecl bundle install https://github.com/hpcc-systems/Visualizer.git</programlisting></para>
+
+    <para>To use the "ecl bundle install &lt;git url&gt;" command, git must be
+    installed on your machine and accessible to the user (in the path).</para>
+
+    <para></para>
+  </sect1>
+
+  <sect1 id="VizUsing">
+    <title>Using the Visualization library</title>
+
+    <para>Once installed, you merely IMPORT the library, then call any method
+    that is appropriate for your data shape.</para>
+
+    <para>For Example:</para>
+
+    <programlisting format="linespecific">IMPORT Visualizer;
+ds := DATASET([ {'English', 5},
+                {'History', 17},
+                {'Geography', 7},
+                {'Chemistry', 16},
+                {'Irish', 26},
+                {'Spanish', 67},
+                {'Bioligy', 66},
+                {'Physics', 46},
+                {'Math', 98}],
+                {STRING subject, INTEGER4 year});
+data_example := OUTPUT(ds, NAMED('Chart2D__test'));
+data_example;
+viz_bubble := Visualizer.TwoD.Bubble('bubble',, 'Chart2D__test');
+viz_bubble;
+</programlisting>
+
+    <para>This code creates a Bubble chart as shown below:</para>
+
+    <graphic fileref="../../images/viz10.jpg" />
+
+    <para> </para>
+
+    <para>See <link linkend="VizMethods">Methods</link> for details about the
+    categories of visualization methods and all available methods.</para>
+
+    <para></para>
+
+    <para></para>
+  </sect1>
+
+  <sect1 id="VizViewing">
+    <title>Viewing the Visualization</title>
+
+    <para>After running a query with a visualization included, you can see the
+    visualization in ECL Watch.</para>
+
+    <para>Open the workunit, then select the <emphasis
+    role="bold">Resources</emphasis> tab.</para>
+
+    <para><graphic fileref="../../images/viz07.jpg" /></para>
+
+    <para></para>
+  </sect1>
+
+  <sect1 id="VizDermProp">
+    <title>Dermatology Properties</title>
+
+    <para>The dermatology layer provides a means of setting properties for the
+    manner in which a visualization appears. </para>
+
+    <para>For example, you can change the palletteID to a new color scheme.
+    After running a query with a visualization included, you can see the
+    visualization in ECL Watch.</para>
+
+    <para>You can set properties in your ECL code or afterwards in ECL Watch:
+    </para>
+
+    <orderedlist>
+      <listitem>
+        <para>Open the workunit, then select the <emphasis
+        role="bold">Resources</emphasis> tab.</para>
+      </listitem>
+
+      <listitem>
+        <para>Press the <emphasis role="bold">Properties</emphasis> button.
+        Open the workunit, then select the <emphasis
+        role="bold">Resources</emphasis> tab.</para>
```

Comment:
Duplicate text from previous listitem:  "Open the workunit, then select the <emphasis role="bold">Resources</emphasis> tab.</para>"